Mail/Views/Menu Drawer/MenuHeaderView.swift

```diff
@@ -16,40 +16,68 @@
  along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
 
+import InfomaniakCore
+import InfomaniakDI
 import MailCore
 import MailCoreUI
 import MailResources
 import SwiftUI
 import UIKit
 
 struct MenuHeaderView: View {
+    @LazyInjectService private var platformDetector: PlatformDetectable
+
     @EnvironmentObject private var mainViewState: MainViewState
 
+    private var menuDrawerLogoHeight: CGFloat {
+        platformDetector.isMac ? UIConstants.menuDrawerLogoMacOSHeight : UIConstants.menuDrawerLogoHeight
+    }
+
     var body: some View {
-        HStack {
-            MailResourcesAsset.logoText.swiftUIImage
-                .resizable()
-                .scaledToFit()
-                .frame(height: UIConstants.menuDrawerLogoHeight)
-
-            Spacer()
-
-            Button {
-                mainViewState.settingsViewConfig = SettingsViewConfig(baseNavigationPath: [])
-            } label: {
-                IKIcon(MailResourcesAsset.cog, size: .large)
+        if platformDetector.isMac {
+            ZStack {}
```

Comment:
Is it possible to put an EmptyView() instead of a ZStack {}?

Reply:
If use EmptyView, toolbar seems to be ignored